students/aravichander/lesson07/Mailroom4.py

```diff
@@ -0,0 +1,88 @@
+#Mailroom #2
```

Comment:
what's new here?

be sure to let us know what you've done, so we can review it.